src/tests/libecalc/input/mappers/variables_mapper/test_variables_mapper.py

```diff
@@ -9,8 +9,6 @@
 )
 from libecalc.presentation.yaml.yaml_types.yaml_variable import YamlSingleVariable
 
-# from libecalc.presentation.yaml.yaml_types import YamlSingleVariable
```

Comment:
YamlVariables still do not use YamlExpressionType, this is because we expect the type to be parsed into an expression. YamlExpressionType only validates. We should look into parsing into an expression in YamlExpressionType I think, if we go that route we probably don't need to implement pydantic specific methods in expression.py (`__get_pydantic_core_schema__` etc.). The alternative is to make expression.py validate similar to YamlExpressionType and use that in the Yaml classes.